script.js

```diff
@@ -0,0 +1,50 @@
+class Todo {
+  constructor() {
+    this.id = 0;
```

Comment:
will be better to name `lastId`, because Todo instance does not have an id, only separate todos